storage/mongo/index.js

```diff
@@ -1 +1,5 @@
-// JavaScript source code
+var MongoUtil = require('utils.js');
+
+module.exports = function(config){
+    new MongoUtil(config).init();
```

Comment:
There is no need to create an object of MongoUtil you can directly access it as a MongoUtil

Reply:
Here you are directly exporting a single function in module.exports. moving forward when we will add function definitions then this wont work. Please add init as a separate function here and export in moudle.exports